microsoft-intune/operations.py

```diff
@@ -0,0 +1,574 @@
+            'description': params.get('description'),
+            'deviceCompliance': convert_string_to_lowercase(params.get('deviceCompliance')),
+            'fileEncryptionLevel': File_Encryption_Level.get(params.get('fileEncryptionLevel')) if params.get(
+                'fileEncryptionLevel') else '',
+            'fileSharingSaveAs': convert_string_to_lowercase(params.get('fileSharingSaveAs')),
+            'managedBrowser': convert_string_to_lowercase(params.get('managedBrowser')),
+            'offlineWipeTimeout': params.get('offlineWipeTimeout'),
+            'pin': convert_string_to_lowercase(params.get('pin')),
+            'pinNumRetry': params.get('pinNumRetry'),
+            'touchId': convert_string_to_lowercase(params.get('touchId'))
+        }
+    }
+    payload = check_payload(payload)
+    response = api_request("POST", url, connector_info, config, data=json.dumps(payload))
+    return response
+
+
+def delete_ios_policy(config, params, connector_info):
+    url = "/providers/Microsoft.Intune/locations/{0}/iosPolicies/{1}?api-version=2015-01-14-preview".format(
+        params.get('hostName'), params.get('policyName'))
+    response = api_request("DELETE", url, connector_info, config)
+    return response
+
+
+def add_app_to_ios_policy(config, params, connector_info):
+    url = "/providers/Microsoft.Intune/locations/{0}/iosPolicies/{1}/apps/{2}?api-version=2015-01-14-preview".format(
+        params.get('hostName'), params.get('policyName'), params.get('appName'))
+    response = api_request("POST", url, connector_info, config)
+    return response
+
+
+def get_apps_for_ios_policy(config, params, connector_info):
+    url = "/providers/Microsoft.Intune/locations/{0}/iosPolicies/{1}/apps?api-version=2015-01-14-preview".format(
+        params.get('hostName'), params.get('policyName'))
+    filter = params.get('$filter')
+    if filter:
+        filter = 'properties/' + filter
+    select = params.get('$select')
+    if select:
+        select = 'properties/' + select
+    payload = {
+        '$filter': filter,
+        '$select': select,
+        '$top': params.get('$top')
+    }
+    payload = check_payload(payload)
+    response = api_request("GET", url, connector_info, config, params=payload)
+    return response
+
+
+def delete_app_for_ios_policy(config, params, connector_info):
+    url = "/providers/Microsoft.Intune/locations/{0}/iosPolicies/{1}/apps/{2}?api-version=2015-01-14-preview".format(
+        params.get('hostName'), params.get('policyName'), params.get('appName'))
+    response = api_request("DELETE", url, connector_info, config)
+    return response
+
+
+def add_group_to_ios_policy(config, params, connector_info):
+    url = "/providers/Microsoft.Intune/locations/{0}/iosPolicies/{1}/groups/{2}?api-version=2015-01-14-preview".format(
+        params.get('hostName'), params.get('policyName'), params.get('groupId'))
+    response = api_request("POST", url, connector_info, config)
```

Comment:
Good coding practice: use constants for hardcoded string if its used at multiple places, i.e. POST, GET, etc.

Reply:
same with diff params